conf/scylla.yaml

```diff
@@ -94,6 +94,9 @@ seed_provider:
           # For example: "<IP1>,<IP2>,<IP3>"
```

Comment:
Again, a better patch description would be:

```
db/config: add endpoint_expiration_time

This commit only adds the config item, it is not used yet. It will be propagated to gossip in the next patch.
```